From 23c5007fe3d10f30fd12cdf605ef23f446beef86 Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@stroblindustries.com>
Date: Fri, 11 Jan 2019 22:24:35 +0200
Subject: [PATCH 1/1] Revert upstream breakage due to schema change.

This reverts commit 558502d379592bef370baf5f246872c84ee930a2. Frankly, this never should've happened. I'm not really sure what upstream was thinking, switching to schemas they weren't even going to ship in their "toolkit" rather than ones which can be reliably found in the GNOME platform.

This change broke Tootle. Not happy.
---
 lib/DateTime.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/DateTime.vala b/lib/DateTime.vala
index 2069e1f..2df9c76 100644
--- a/lib/DateTime.vala
+++ b/lib/DateTime.vala
@@ -110,7 +110,7 @@ namespace Granite.DateTime {
      * @return true if the clock format is 12h based, false otherwise.
      */
     private static bool is_clock_format_12h () {
-        var h24_settings = new Settings ("io.elementary.desktop.wingpanel.datetime");
+        var h24_settings = new Settings ("org.gnome.desktop.interface");
         var format = h24_settings.get_string ("clock-format");
         return (format.contains ("12h"));
     }
-- 
2.20.1

